llvm-bazel/llvm-project-overlay/llvm/unittests/BUILD

```diff
@@ -465,6 +465,9 @@ cc_test(
         "$(STACK_FRAME_UNLIMITED)",
     ],
     linkstatic = 1,
+    tags = [
+        "local", # Not compatible with the sandbox on MacOS
+    ],
```

Comment:
Just note that this appears to make the test take about 10 times as long. Would it be better to disable the gid/uid test entirely with `--gtest_filter`?